<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalizerClass</class>
 <widget class="QMainWindow" name="LocalizerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Localizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QGroupBox" name="analysisGroupBox">
      <property name="minimumSize">
       <size>
        <width>561</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Hilbert Transform Analysis</string>
      </property>
      <widget class="QTabWidget" name="freqTabWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>531</width>
         <height>291</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="sameanaCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>331</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Execute the same Analysis for each Frequency Band</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="FolderCountLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QTextBrowser" name="messageDisplayer">
      <property name="minimumSize">
       <size>
        <width>815</width>
        <height>114</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="processButton">
        <property name="minimumSize">
         <size>
          <width>87</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Process Analysis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="minimumSize">
         <size>
          <width>87</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel Analysis</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="DeselectableTreeView" name="FileTreeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>241</width>
        <height>520</height>
       </size>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionCharger_un_dossier_Patient"/>
    <addaction name="actionCharger_un_fichier_individuel"/>
   </widget>
   <widget class="QMenu" name="menuOutils">
    <property name="title">
     <string>Outils</string>
    </property>
    <addaction name="actionTRSecator"/>
    <addaction name="actionConcatenator"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionPerformance"/>
    <addaction name="actionStats"/>
    <addaction name="actionPictures"/>
    <addaction name="actionLocalizers"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuOutils"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCharger_un_dossier_Patient">
   <property name="text">
    <string>Charger un dossier Patient</string>
   </property>
  </action>
  <action name="actionCharger_un_fichier_individuel">
   <property name="text">
    <string>Charger un fichier individuel</string>
   </property>
  </action>
  <action name="actionTRSecator">
   <property name="text">
    <string>TRSecator</string>
   </property>
  </action>
  <action name="actionConcatenator">
   <property name="text">
    <string>Concatenator</string>
   </property>
  </action>
  <action name="actionStats">
   <property name="text">
    <string>Statistic</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionPerformance">
   <property name="text">
    <string>Performance </string>
   </property>
  </action>
  <action name="actionPictures">
   <property name="text">
    <string>Pictures</string>
   </property>
  </action>
  <action name="actionLocalizers">
   <property name="text">
    <string>Localizers</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DeselectableTreeView</class>
   <extends>QTreeView</extends>
   <header>DeselectableTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="localizer.qrc"/>
 </resources>
 <connections/>
</ui>
